src/Transport/Entities/FavouritePlaceIcon.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Base\Transport\Entities;
+
+class FavouritePlaceIcon
+{
+    private $category = '';
+    private $label = '';
+
+    /** @return string */
+    public function getCategory()
+    {
+        return $this->category;
+    }
+
+    /** @param string $category */
+    public function setCategory($category)
+    {
+        $this->category = strtolower($category);
```

Comment:
We're saying $category has to be a certain format. Why is that? Should we then remove  `[^\w]` too?